src/entity_ref.rs

```diff
@@ -178,23 +217,62 @@ impl<'a, T: Component> RefMut<'a, T> {
     }
 }
 
-unsafe impl<T: Component> Send for RefMut<'_, T> {}
-unsafe impl<T: Component> Sync for RefMut<'_, T> {}
+unsafe impl<T: ?Sized + Send> Send for RefMut<'_, T> {}
```

Comment:
As the `RefMut` provides exclusive access, I've marked the `Send` impl as just requiring the `T` to be `Send`, but I'm not sure about this :-/

Reply:
`RefMut` also provides shared access via `Deref`, so it should be possible to usefully share a `&RefMut<T>` between threads iff `T: Sync`.

Reply:
Er, but this is the `Send` impl. I think this is correct.